### ISSUE TITLE
Test state-preserving OpenES 0.10→0.08 learning-rate schedules

### ISSUE THREAD

**Issue by Ahmadfauzi34** [Ahmadfauzi34/ai_agent_builder#276]
## Purpose

Use the state-preserving OpenES learning-rate control merged in #275 to test the schedule hypothesis established by #256/#257.

This is a host-owned research experiment. It must not add automatic scheduling policy to the Rust core.

## Evidence for bounded transition points

Fixed-rate refinement showed:

| Generation | LR 0.08 | LR 0.10 | Leader |
| --- | ---: | ---: | --- |
| 48 | -0.552942 | **-0.489408** | 0.10 |
| 96 | -0.354409 | **-0.330293** | 0.10 vs 0.08 |
| 144 | **-0.307413** | -0.308263 | 0.08 |
| 192 | **-0.303202** | -0.304647 | 0.08 |

The useful crossover is therefore bounded between generations 96 and 144.

## Fixed workload

Keep the #257 lineage unchanged:

- installed Python wheel + typed host;
- policy `Linear(6 -> 8) -> ReLU -> Linear(8 -> 2)`;
- parameter dimension 74;
- seed `20_260_917`;
- population 8;
- sigma 0.08;
- 192 generations;
- deterministic four-scenario, 16-step control task;
- 1536 candidate evaluations per case.

## Matrix

```text
FIXED_010        LR 0.10 for generations 1..192
FIXED_008        LR 0.08 for generations 1..192
SCHEDULE_G96     LR 0.10 through generation 96, then 0.08
SCHEDULE_G144    LR 0.10 through generation 144, then 0.08
```

For scheduled cases, call `set_learning_rate(0.08)` only after the transition generation's `tell()` has completed and before the next `ask_f32()`.

## Continuity proof

A schedule must prove it is the same optimizer trajectory, not a restart:

1. generation counter is unchanged by the setter;
2. champion/best parameters are unchanged by the setter;
3. graph and binding identities remain unchanged;
4. the first post-transition `ask_f32()` bytes are exactly equal to the corresponding fixed-LR-0.10 run at the same generation;
5. divergence is allowed only after that identical candidate batch is consumed by `tell()` at LR 0.08.

This binds the research directly to the continuity contract proven in #275.

## Evidence

For every case retain the same measurements as #257:

- champion rewards at 48/96/144/192;
- final reward;
- improvement generations / plateau length / last improvement;
- interval improvement counts;
- fitness std;
- diversity;
- normalized antithetic contrast;
- mean-norm path length;
- exact candidate count;
- stable program/binding identities.

Also record:

- LR transition generation;
- pre/post-transition reported LR;
- exact first-post-transition candidate-byte equality versus FIXED_010;
- state-continuity checks at the transition.

Replay the highest final-reward case exactly through stateful ProgramBundle.

## Interpretation

- a schedule beats both fixed controls at generation 192 -> schedule hypothesis supported;
- schedule matches the better fixed control while improving early milestones -> useful schedule without final penalty;
- both schedules underperform fixed controls -> fixed LR remains preferable for this workload;
- G96 beats G144 -> earlier refinement is favored;
- G144 beats G96 -> preserve high LR longer.

No production setting is promoted from one deterministic workload.

## Scope

Research-only.

No new core optimizer logic.
No automatic schedule.
No sigma mutation.
No ABI change.
No graph/Math/ProgramBundle semantic change.
No support-matrix widening.
